### PR TITLE
plugin(xml-threat) adds new plugin documentation

### PR DESCRIPTION
**Pull request by Tieske**
### Summary
Adds new xml-threat-protection plugin

### CODE REVIEW

**Comment by cloudjumpercat** [comment]
@Tieske Thanks for submitting this! Since this is for Gateway 3.1, could you change the base branch to `release/gateway-3.1`? Thanks!

**Comment by Tieske** [comment]
@cloudjumpercat updated. Let me know if you need anything else.

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]
Left a few copyedits

File: app/_hub/kong-inc/xml-threat-protection/_index.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+      datatype: boolean
+      description: |
+        Indicating whether an XML Document Type Definition (DTD) section is allowed.
+    - name: namespace_aware
+      required: true
+      default: true
+      value_in_examples: null
+      datatype: boolean
+      description: |
+        If not parsing namespace aware, all prefixes and namesapce attributes will be counted as regular attributes and element names, and validated as such.
```
[the remark this answers is not in the document]
Comment (cloudjumpercat) [reply app/_hub/kong-inc/xml-threat-protection/_index.md:51]: Not sure if my revision is accurate or not.

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Pull request merged**